scripts/aem.js

```diff
@@ -699,6 +699,9 @@ function decorateBlocks(main) {
  * @returns {Promise}
  */
 async function loadHeader(header) {
```

Comment:
@akki761 We can not make any changes in aem.js, it;s core library like /libs in aem

Reply:
Also, i am thiking to change this rule to check if First div inside main has hero-banner class, this will allow us to have overlaying banner on any page on future without breaking.

<img width="461" alt="image" src="https://github.com/AEM-COMP/aem-competency/assets/129100763/cf82222c-490a-43d9-ab68-74aa15b0cea6">